ansible_navigator/configuration_subsystem/configurator.py

```diff
@@ -83,15 +83,16 @@ def configure(self) -> Tuple[List[LogMessage], List[str]]:
         restore them
         """
         self._config.original_command = self._params
-        message = f"Params provided: {self._config.original_command}"
-        self._messages.append(LogMessage(level=logging.DEBUG, message=message))
+        cmd_message = f"Command provided: '{' '.join(self._config.original_command)}'"
```

Comment:
Is this enough? What will this do if the original command has an argument with spaces in it? I think that information will be lost. (i.e. args `foo --bar "baz buzz" biz` will get printed as `foo --bar baz buzz biz`)... Unless I'm wrong. :)

Reply:
just testing this out, sys.argv handles the quotes when it generates the list
```
➜  /tmp python go.py foo --bar "baz buzz" biz
['go.py', 'foo', '--bar', 'baz buzz', 'biz']
```

but in the actions we just split the `:` command:

```
>>> foo = 'foo --bar "baz buzz" biz'
>>> foo.split()
['foo', '--bar', '"baz', 'buzz"', 'biz']
```
https://github.com/ansible/ansible-navigator/blob/c4957f0228303a6baeb07945635229c6ac18baed/ansible_navigator/actions/run.py#L308

but we _should_ be using shlex

```
>>> import shlex
>>> shlex.split(foo)
['foo', '--bar', 'baz buzz', 'biz']
```

https://github.com/ansible/ansible-navigator/issues/195

I'll change this to shlex.join in anticipation of that

Reply:
nice catch, btw, could have been a nasty little bug later on